src/common/components/MessageInput/index.tsx

```diff
@@ -65,7 +65,7 @@ const MessageInput = ({ chatId, isSupport = true }: MessageInputProps) => {
           autoFocus
           {...register("message")}
           multiline
-          minRows={5}
+          minRows={2}
```

Comment:
5 rows looks better on dashboard, perhaps use a variable with default value of 5 and change to 2 in chat pop up.

Reply:
Just have a prop that sets the mode of the chat whether its the popup window or not. This should be resolved within this PR.